manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Luca-AlexanerThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+PackageLocale: de-DE
+Publisher: Luca-Alexander Thomas
+PublisherUrl: https://lucathomas.de
+PublisherSupportUrl: https://lucathomas.de/kontakt
+PrivacyUrl: https://lucathomas.de/datenschutzerklaerung/
+Author: Luca-Alexander Thomas
+PackageName: AutoHome-Autostart

```

This needs to match **A**dd or **R**emove **P**rograms (ARP)
```suggestion
PackageName: AutoHome
```